### ISSUE TITLE
[FR] Display outside temperature in header

### ISSUE THREAD

**Issue by rkotulan** [dwainscheeren/lovelace-dwains-theme#19]
Display outside temperature in the header when you setup outside_temperature in global.yaml

[[[ 
      if(states['{{ _d_t_config.global["outside_temperature"] }}']){
        {% if _d_t_config.global["outside_temperature"].split('.')[0] == 'sensor' %}
          return (Math.round(states['{{ _d_t_config.global["outside_temperature"] }}'].state * 10) / 10) + '° {{ _d_t_trans.climate.outside }}';
        {% elif _d_t_config.global["outside_temperature"].split('.')[0] == 'climate' %}
          return states['{{ _d_t_config.global["outside_temperature"] }}'].attributes.current_temperature + '° {{ _d_t_trans.climate.outside }}'; 
        {% else %}
          return 'NaN';
        {% endif %}
      }  else {
        if(entity){
          return entity.attributes.temperature + "° {{ _d_t_trans.climate.outside }}";
        } else {
          console.log('Dwains Theme Error: Invalid global.yaml:weather entity!');
          return 'Weather Entity Error!';
        }
      }
    ]]]

**Comment by dwainscheeren** [comment]
Yeah I used the weather integration outside temperature, but maybe somebody has a real outdoor temperature sensor indeed. Maybe its better to do it like this then. I will use it in next update, thanks.
